README.md

```diff
@@ -97,7 +97,7 @@ See [variables.tf] and [examples/] for details and use-cases.
 
   Default is `{}`.
 
-- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(any)`)*<a name="var-module_depends_on"></a>
+- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(object)`)*<a name="var-module_depends_on"></a>
```

Comment:
how did that happen? please update from template ;)

Reply:
Terradoc handling things differently,, Have updated it to be correct.